jre/checks/oval/installed_app_is_java.xml

```diff
@@ -69,4 +70,10 @@
   <linux:rpminfo_state id="state_java_version" version="1">
     <linux:version datatype="version" operation="equals">1.8.0</linux:version>
   </linux:rpminfo_state>
+  <linux:rpminfo_state id="state_java_version_minimum" version="1">
+    <linux:version datatype="version" operation="greater than or equal">1.8.0</linux:version>
+  </linux:rpminfo_state>
+  <linux:rpminfo_state id="state_java_version_maximum" version="1">
+    <linux:version datatype="version" operation="less than">1.9.0</linux:version>
```

Comment:
@lordofhyphens Does the check as proposed work for you?

Checking the OVAL standard for `datatype="version"`, it says that "[short numbers] should be padded [to match length]".
Reference: https://oval-community-guidelines.readthedocs.io/en/latest/oval-schema-documentation/oval-common-schema.html#simpledatatypeenumeration
But I don't know if OpenSCAP scanner does that automatically, or the content author should do it.
@evgenyz @jan-cerny Do you, by chance, know something about this?

A package version of  `1.8.0_281` would be equivalent of `1.8.0.281`, which would still be greater than `1.8.0.0` and smaller than `1.9.0.0`, if automatically padded by the scanner.

Reply:
@yuumasato I've verified that this does indeed work. As you mentioned, the addition of the `_XXX` just added an additional padding to the `1.8.0` original check which is why it failed.. It would be so much nicer if the OVAL check would simply check what you supplied it to check - ie `1.8.0` instead of adding an additional padded number! Kind of like a subset check.